pkg/internal/controller/controller_test.go

```diff
@@ -400,8 +379,10 @@ var _ = Describe("controller", func() {
 		// TODO(directxman12): we should ensure that backoff occurrs with error requeue
 
 		It("should not reset backoff until there's a non-error result", func() {
-			dq := &DelegatingQueue{RateLimitingInterface: ctrl.NewQueue("controller1", nil)}
-			ctrl.NewQueue = func(string, ratelimiter.RateLimiter) workqueue.RateLimitingInterface { return dq }
+			dq := &DelegatingQueue{TypedRateLimitingInterface: ctrl.NewQueue("controller1", nil)}
```

Comment:
Also wondering here if this should stay "RateLimitingInterface" as it's pinned to reconcile.Request